tcs-persistence/src/test/java/com/transformuk/hee/tis/tcs/service/repository/ProgrammeRepositoryIntTest.java

```diff
@@ -63,6 +63,7 @@ public void findByCurriculumMembershipPersonIdShouldReturnAUniqueCollectionOfPro
     personRepository.saveAll(Lists.newArrayList(person1, person2));
     personRepository.flush();
 
+    //TODO: rewrite with new data structure
```

Comment:
Yes, I think so. The issue is that the method under test 'findByCurriculumMembershipPersonId()' uses fields in the curriculum membership table that will no longer be populated (or in fact, exist) in future. I think I thought it would be better to rewrite the test when we make that change, since at this point we still maintain the duplication of fields between CM and PM.

Reply:
I agree its not a very useful / clear comment though 🤦